pkg/core/localize/locales/en/cmd/rule.en.toml

```diff
@@ -85,95 +86,95 @@ one = '"{{.Type}}" rule not enabled for artifact with ID "{{.ID}}"'
 one = '"{{.Type}}" rule not enabled for Service Registry instance with ID "{{.ID}}"'
 
 [registry.rule.common.error.conflict]
-one = 'rule of type "{{.Type}}" is already enabled'
+one = 'Rule of type "{{.Type}}" is already enabled'
 
 [registry.rule.enable.log.info.ruleEnabled]
 one = 'Rule successfully enabled'
 
 [registry.rule.list.cmd.description.short]
-one='List validity and compatibility rules'
+one='List the validity and compatibility rules'
 
 [registry.rule.list.cmd.description.long]
-one='View validity and compatibility rules for the specified Service Registry instance or artifact.'
+one='List the validity and compatibility rules for the specified Service Registry instance or artifact.'
 
 [registry.rule.list.cmd.example]
 one='''
-## List global rules for artifacts of the current Service Registry instance
+## List the global rules for all artifacts in the current Service Registry instance
 $ rhoas service-registry rule list
 
-## List global rule for artifacts of a specific Service Registry instance
-$ rhoas service-registry rule list --instance-id 8ecff228-1ffe-4cf5-b38b-55223885ee00
+## List the global rules for all artifacts in a specific Service Registry instance
+$ rhoas service-registry rule list --instance-id=8ecff228-1ffe-4cf5-b38b-55223885ee00
 
-## List rule for a specific artifact
+## List the artifact-specific rules for a particular artifact
```

Comment:
Suggest "List the artifact rules for a specific artifact" for minimalism. Don't think we need "artifact-specific" and "particular artifact" in the same sentence.

Reply:
Done :)